dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -75,7 +75,6 @@ def teardown
       user_id: @student.id,
       submitted: 'true'
     )
-    Metrics::Events.stubs(:log_event).never
```

Comment:
Statsig will run in local_mode when in tests and won't log or send events. I think these can all be removed in these unit tests.